public/dark.css

```
@@ -0,0 +1,151 @@
+  color: var(--light-font-color);
+}
+
+code.language-terminal {
+  color: var(--dark-font-color);
+}
+
+[type='search'] {
+  border: 1px solid var(--light-background);
+  background: var(--light-background);
+  color: var(--dark-font-color);
+  border: 2px solid transparent;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)